apps/base-docs/docs/building-with-base/guides/build-with-cookbook.md

```diff
@@ -0,0 +1,341 @@
+- Cookbook's Remix IDE Plugin-in
+- Cookbook's Hardhat Boilerplate
+- Cookbook's Foundry Boilerplate
+
+We will be using [Cookbook's Simple ERC-20 Token Smart Contract](https://www.cookbook.dev/contracts/simple-token?utm=basedocs) throughout this guide, but the principles here will apply to any smart contract found on or imported to [Cookbook](https://www.cookbook.dev?utm=basedocs). 
+
+## Objectives
+
+By the end of this guide you should be able to do the following:
+
+- Search Cookbook for Base protocols and smart contracts
+- Import smart contract code into your preferred development environment
+- Compile a smart contract for Base
+- Deploy a smart contract to Base
+- Interact with a smart contract deployed on Base
+
+## Prerequisites
+
+## Coinbase Wallet
+
+In order to deploy a smart contract, you will first need a web3 wallet. You can create a wallet by downloading the Coinbase Wallet browser extension.
+
+* Download [Coinbase Wallet](https://chrome.google.com/webstore/detail/coinbase-wallet-extension/hnfanknocfeofbddgcijnmhnfnkdnaad?hl=en)
+
+## Wallet funds
+Deploying contracts to the blockchain requires a gas fee. Therefore, you will need to fund your wallet with ETH to cover those gas fees.
+
+For this guide, you will be deploying a contract to the Base Goerli test network. You can fund your wallet with Base Goerli ETH using the following options:
+
+* [Coinbase Faucet | Base Goerli](https://www.coinbase.com/faucets/base-ethereum-goerli-faucet)
+* [Coinbase Wallet Faucets](https://chrome.google.com/webstore/detail/coinbase-wallet-extension/hnfanknocfeofbddgcijnmhnfnkdnaad)
+
+For more detailed steps on funding your wallet with Base Goerli ETH, see [Network Faucets](https://docs.base.org/tools/network-faucets/).
+
+# Search Cookbook's Smart Contract Registry
+
+Navigate to [cookbook.dev/chains/Base](https://www.cookbook.dev/chains/Base?utm=basedocs) and explore **Protocols** on Base, or search for specific smart contracts in the search bar. 
+
+![Search Base Protocols](../../../assets/images/deployment-with-cookbook/Search-Base-Protocols.png)
+
+To learn about a smart contract on Cookbook, select the contract, and select `Expand`. This opens the code alongside ChefGPT, Cookbook's AI Solidity assistant. 
+
+Highlight selections of the code and press **Analyze Snippet** to get more information about the smart contract code you're looking at, or ask ChefGPT questions about Base, solidity, or your smart contract.
+
+![Analyze Code Snippet](../../../assets/images/deployment-with-cookbook/Analyze-Snippet.png)
+
+## Import any Smart Contract Code into Cookbook
+
+Import verified smart contract code into Cookbook to fork, learn about, or build with by inputting any smart contract address into the Cookbook search bar.
+
+![Cookbook Import Contract](../../../assets/images/deployment-with-cookbook/Import-Contract.png)
+
+:::tip
+Supports Base, Ethereum, Polygon, BSC, Fantom, Optimism, Arbitrum, Moonbeam, Moonriver, Gnosis and Celo
```

Comment:
Consider removing this tip admonition. Although it's cool it supports all of these other networks, I don't think it's necessary to call out for this doc.

Reply:
Thanks for reviewing. I've updated the documentation to reference and work with Base Sepolia rather than Base Goerli. I agree with your thoughts regarding the tip and have removed it.